### PR TITLE
added files to step users through building cake on ios

### PR DESCRIPTION
**Pull request by tannerdsilva**

### CODE REVIEW

**Comment by mkyq** [comment]
`Intel Mac` - it should works same on arm too. 
`- Android SDK 28
  - Android NDK 17c`
  I'm not sure that android sdk and ndk are required for iOS build :)

**Comment by mkyq** [comment]
And please add that need to checkout v4.2.7 tag after git clone

**Comment by mkyq** [comment]
I just updated main with a lot of changes. Now need to update the instruction: 
before `$ flutter packages pub run build_runner build --delete-conflicting-outputs` in root app directory need to do next:
- `cd cw_core/` + `$ flutter packages pub run build_runner build --delete-conflicting-outputs`
(then cd ..) 
- `cd cw_monero/` + `$ flutter packages pub run build_runner build --delete-conflicting-outputs`

Also now need to generate different info.plist file and icon for monero.com and cakewallet apps. For that need to:
(in root directory)
- `cd scriptips/ios`
- `./app_env <monero.com OR cakewallet>` based on what type of the app you want to build
- `./app_config.sh`
(back to root directory)

Steps above can be done before call `flutter run` command.

**Comment by tannerdsilva** [comment]
@mkyq pushed updates

File: howto-build-ios.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# Building CakeWallet for iOS
+
+## Requirements and Setup
+
+The following are the system requirements to build CakeWallet for your iOS device.
+
+```
+Intel Mac
```
Comment (mkyq) [inline howto-build-ios.md:8]: This instruction should works well on Apple arm too.

File: howto-build-ios.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+Xcode
+Homebrew
+Flutter 2 or above
```
Comment (mkyq) [inline howto-build-ios.md:11]: Need Flutter 2.0.4

File: howto-build-ios.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+	- Android SDK 28
+	- Android NDK 17c
```
Comment (mkyq) [inline howto-build-ios.md:13]: Is unnecessary for iOS

**Review by mkyq — CHANGES_REQUESTED** [review]

**Pull request closed without merging**